pkg/project/resource_project_repo_test.go

```diff
@@ -114,3 +116,128 @@ func TestAccProjectRepo(t *testing.T) {
 		},
 	})
 }
+
+/*
+Test to assign large number of repositories to a project
+*/
+func TestAccAssignMultipleReposInProject(t *testing.T) {
+
+	const numRepos = 100
+	const repoNameInitial = "repo-"
+
+	name := "tftestprojects" + randSeq(10)
+	resourceName := "project." + name
+	projectKey := strings.ToLower(randSeq(6))
+
+	repoNamesStr := func(repoCount int) string {
+		var repoNames []string
+		for i := 0; i < repoCount; i++ {
+			repoNames = append(repoNames, repoNameInitial+strconv.Itoa(i))
```

Comment:
I want the end value of repoNamesStr will look like
```
["repo-0","repo-1","repo-2",...,"repo-198","repo-199"]
```
then it can be passed on to the template

Reply:
As I mentioned before, `json.Marshal()` with a slice of strings will work equally well and much clearer what it does.

Reply:
Current code is
```
repoNamesStr := func(repoCount int) string {
		var repoNames []string
		for i := 0; i < repoCount; i++ {
			repoNames = append(repoNames, repoNameInitial+strconv.Itoa(i))
		}
		jsonStr, err := json.Marshal(repoNames)
		if err != nil {
			return "[]"
		}
		return fmt.Sprint(string(jsonStr))
	}
```
expected end value  of `repoNamesStr` is
```
["repo-0","repo-1","repo-2",...,"repo-198","repo-199"]
```